.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Test deployment
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  test-deploy:
+    name: Test deployment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+          cache-dependency-path: src/yarn.lock
+
+      - name: Install dependencies
+        run: yarn --cwd src install --frozen-lockfile
```

Comment:
Can you teach me about the benefit of `--frozen-lockfile` here @andresatvery?

Reply:
Sure! Basically when you do a `yarn install` the way it works is that it might modify the lock file to add dependencies, adding the `--frozen-lockfile` makes it that whenever it wants to add a dependency to the lock file it will warn, so in this case, the build will fail and we need to make sure the dependencies are correct in development

For more context and better examples check this: https://github.com/yarnpkg/yarn/issues/4147

Reply:
Cool beans, thanks!